gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/LocalGcdHelper.java

```diff
@@ -179,49 +179,134 @@ private static Path executablePath(String cmd) {
   }
 
   private static class ProcessStreamReader extends Thread {
-
-    private final Process process;
     private final BufferedReader reader;
 
-    ProcessStreamReader(Process process, String blockUntil) throws IOException {
+    ProcessStreamReader(InputStream inputStream) {
```

Comment:
Lets pass a string ("STDOUT" or "STDERR") and include it in the string.
e.g.
super("Local GCD " + name + " Inputstream reader")